.github/workflows/test-external-providers.yml

```diff
@@ -45,20 +45,22 @@ jobs:
 
       - name: Build distro from config file
         run: |
-          USE_COPY_NOT_MOUNT=true LLAMA_STACK_DIR=. uv run llama stack build --config tests/external-provider/llama-stack-provider-ollama/custom-distro.yaml
+          USE_COPY_NOT_MOUNT=true LLAMA_STACK_DIR=. llama stack build --config tests/external-provider/llama-stack-provider-ollama/custom-distro.yaml
```

Comment:
Why was this removed? Don't you need a specific uv env?

Reply:
We use the env setup by the setup-runner task, the env is auto activated, we just need the llama binary and then we re-use the venv created by the build command.